WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameViewController.swift

```diff
@@ -0,0 +1,47 @@
+
+import UIKit
+
+/// Site Name screen for the Site Creation flow
+class SiteNameViewController: UIViewController {
+
+    private let creator: SiteCreator
+
+    init(creator: SiteCreator) {
+        self.creator = creator
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func loadView() {
+        // TODO: SITENAME - This string is here only to test the UI until the implementation is complete.
+        view = SiteNameView(siteName: "Commuinity & Non-Profit")
```

Comment:
The nittiest of nits, since this is temporary. 😆 

```suggestion
        view = SiteNameView(siteName: "Community & Non-Profit")
```